plugins/woocommerce/tests/e2e-pw/tests/merchant/product-reviews.spec.js

```diff
@@ -1,63 +1,142 @@
+				consumerSecret: process.env.CONSUMER_SECRET,
+				version: 'wc/v3',
 			} );
-		await api.post( 'products/reviews', {
-			product_id: productId,
-			review: productReview,
-			reviewer: 'John Doe',
-			reviewer_email: reviewerEmail,
-			rating: randomRating,
-		} );
+
+			await use( api );
+		},
+		testData: async ( { api }, use ) => {
+			const timestamp = Date.now().toString();
+			let product = {};
+			let review = {};
+
+			// Create the product
+			await api
+				.post( 'products', {
+					name: `Review me ${ timestamp }`,
+					type: 'simple',
+					regular_price: '9.99',
+				} )
+				.then( ( response ) => {
+					product = response.data;
+				} );
+
+			// Create the product review
+			await api
+				.post( 'products/reviews', {
+					product_id: product.id,
+					review: `Nice one, Playwright! ${ timestamp }`,
+					reviewer: 'John Doe',
+					reviewer_email: `john.doe.${ timestamp }@example.com`,
+					rating: ( Math.random() * ( 5 - 1 ) + 1 ).toFixed( 0 ),
+				} )
+				.then( ( response ) => {
+					review = response.data;
+				} );
+
+			await use( { product, review } );
+
+			// Cleanup
+			await api.delete( `products/reviews/${ review.id }`, {
+				force: true,
+			} );
+			await api.delete( `products/${ product.id }`, {
+				force: true,
+			} );
+		},
+	} );
+
+	test( 'can view product review', async ( { page, testData } ) => {
+		await page.goto(
+			`wp-admin/edit.php?post_type=product&page=product-reviews`
+		);
+		await expect(
+			page.getByRole( 'cell', { name: testData.review.review } )
+		).toBeVisible();
+		await expect(
+			page.getByRole( 'link', { name: testData.product.name } )
+		).toBeVisible();
+		await expect(
+			page.getByLabel( `${ testData.review.rating } out of 5` )
+		).toBeVisible();
+		await expect(
+			page.getByRole( 'link', { name: testData.review.reviewer_email } )
+		).toBeVisible();
 	} );
 
-	test.afterAll( async ( {} ) => {
-		await api.delete( `products/${ productId }`, {
-			force: true,
-		} );
+	test( 'can edit product review', async ( { page, testData } ) => {
+		await page.goto(
+			`wp-admin/edit.php?post_type=product&page=product-reviews`
+		);
+		await expect(
+			page.getByRole( 'cell', { name: testData.review.review } )
+		).toBeVisible();
+
+		//Hover over the product review to have 'Edit' displayed
+		await page.hover( '.comment-text' );
+
+		// Create new review, Quick Edit it and save
+		const EditedtimeStamp = Date.now().toString();
+		const updatedReview = `(edited ${ EditedtimeStamp })`;
+		await page.getByRole( 'button', { name: 'Quick Edit' } ).click();
+		await page.locator( '.wp-editor-area' ).first().fill( updatedReview );
+		await page.getByRole( 'button', { name: 'Update Comment' } ).click();
+
+		// Verify that the edited comment is there
+		const commentText = page.locator( '.comment-text' );
+		await commentText.waitFor( { state: 'visible' } );
+		const updatedComment = await commentText.innerText();
+		await expect( updatedComment ).toBe( updatedReview );
```

Comment:
A single line should do the job here.

```suggestion
		await expect( page.getByText( updatedReview ) ).toBeVisible();
```